spec/javascripts/support/jasmine.yml

```diff
@@ -0,0 +1,124 @@
+# src_files
```

Comment:
I wonder if spec/javascript will execute in travis automagically, or if we have to invoke something else.

Generally like jasmine for complex JS testing